packages/console/src/contexts/SubscriptionDataProvider/use-new-subscription-data.ts

```diff
@@ -50,6 +50,14 @@ const useNewSubscriptionData: () => NewSubscriptionContext & { isLoading: boolea
     [currentTenant?.planId, logtoSkus]
   );
 
+  useEffect(() => {
+    if (subscriptionUsageData?.quota) {
+      updateTenant(tenantId, {
+        quota: pick(subscriptionUsageData.quota, 'mauLimit', 'tokenLimit'),
+      });
+    }
+  }, [tenantId, subscriptionUsageData, updateTenant]);
```

Comment:
Narrow down the dependencies list to avoid unnecessary triggers
```suggestion
  }, [tenantId, subscriptionUsageData?.quota, updateTenant]);
```